sweetest/src/main/java/com/mysugr/sweetest/framework/accessor/DependenciesAccessor.kt

```diff
@@ -27,4 +27,8 @@ class DependenciesAccessor(@PublishedApi internal val parent: BaseAccessor) {
     inline fun <reified T : Any> offerMockRequired(noinline initializer: DependencyInitializer<T>) {
         parent.testContext.dependencies.offerMockRequired(T::class, initializer)
     }
+
+    inline fun <reified T : Any> provide() {
```

Comment:
i'd not add it here but rather remove this class as none of it is used anywhere. you can inline the usage of this function, imo